### PR TITLE
ci: add e2e asserting that setup works for users

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: .bazelignore [hunk 1]
```diff
@@ -0,0 +1 @@
+e2e/
```
Comment (gregmagolan) [inline .bazelignore:1]: oh, that is easier than listing them all

File: rollup/dependencies.bzl [hunk 2]
```diff
@@ -0,0 +1,45 @@
+"Fetch dependencies needed by users."
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+
+# WARNING: any changes in this function may be BREAKING CHANGES for users
```
Comment (gregmagolan) [inline rollup/dependencies.bzl:6]: do you mean any new additions?

**Review by gregmagolan — APPROVED** [review]
🌮

File: rollup/dependencies.bzl (hunk 2 above)
Comment (alexeagle) [reply rollup/dependencies.bzl:6]: yeah that's true. This string is everywhere since cloning from an early version of the template :(

Comment (alexeagle) [reply rollup/dependencies.bzl:6]: neeeeeeed bzlmod

**Pull request merged**